### PR TITLE
Use dbExecute() where appropriate

### PR DESCRIPTION
**Pull request by krlmlr**
because dbGetQuery() now gives a warning if used on a statement.

I do wonder why the code uses SQL interpolation instead of the much more efficient parametrized queries.

### CODE REVIEW

File: tests/testthat/test-crashes.R [hunk 1]
```diff
@@ -10,7 +10,7 @@ test_that("requeueing crashed consumers", {
 
   ## now we simulate a crash, so the connection embedded in `msg` is closed
   rm(msg)
-  gc()
+  suppressWarnings(gc())
```
Comment (gaborcsardi) [inline tests/testthat/test-crashes.R]: Why is this needed?

Comment (krlmlr) [inline tests/testthat/test-crashes.R]: Because a dangling open connection will issue a warning when released during garbage collection.

Comment (gaborcsardi) [inline tests/testthat/test-crashes.R]: Got it!

**Comment by gaborcsardi** [comment]
Thanks!

**Comment by gaborcsardi** [comment]
As for parameterized queries, I almost always manipulate a single row, so they are not really needed here.

**Comment by krlmlr** [comment]
They avoid the necessity for parsing the serializing and deserializing values to/from strings, and give opportunity to reuse execution plans (though I don't know about practical implications).
